frontend/src/components/AppBar.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+import { Link } from 'react-router-dom';
+
+import './AppBar.css';
+
+const pages = ['Documents', 'Tags'];
+
+function responsiveAppBar() {
+  return (
+    <AppBar position="static">
+      <Container
+        sx = {{
+          ml: 0,
+        }}
+      >
+        <Toolbar>
+          <Typography
+            variant="h6"
+            noWrap
+            component="a"
+            href="/"
+            sx={{
+              mr: 2,
+              display: 'flex',
+              fontFamily: 'monospace',
+              fontWeight: 700,
+              letterSpacing: '.3rem',
+              color: 'inherit',
+              textDecoration: 'none',
+            }}
+          >
+            HOME
+          </Typography>
+          <Box sx={{ flexGrow: 1, display: 'flex' }}>
+            {pages.map((page) => (
+              <Button
+                key={page}
+                component={Link}
+                to={`/${page}`}
+                sx={{ my: 2, color: 'white', display: 'block' }}
+              >
+                {page}
+              </Button>
+            ))}
+          </Box>
+        </Toolbar>
+      </Container>
+    </AppBar>
+  );
+}
+export default responsiveAppBar;
```

Comment:
also a nit, but I personally prefer doing

```ts
export default function responsiveAppBar() {
    ...
}
```
I think it makes for cleaner code, especially on more complex pages where you have to deal with a bunch of helper functions